cmip6_downscaling/data/cmip.py

```diff
@@ -163,38 +163,38 @@ def cmip():
     return model_dict, data
 
 
-def load_cmip_dictionary(
-    activity_ids=["CMIP"],
-    experiment_ids=["historical"],  # , "ssp126", "ssp245",  "ssp585"
-    member_ids=["r1i1p1f1"],
-    source_ids=["MIROC6"],  # BCC-CSM2-MR"]
-    table_ids=["day"],
-    grid_labels=["gn"],
-    variable_ids=["tasmax"],
-    return_type='zarr',
-):
-    """Loads CMIP6 GCM dataset dictionary based on input criteria.
+def load_cmip(
+    activity_ids: list = ["CMIP"],
+    experiment_ids: list = ["historical"],
+    member_ids: list = ["r1i1p1f1"],
+    source_ids: list = ["MIROC6"],
+    table_ids: list = ["day"],
+    grid_labels: list = ["gn"],
+    variable_ids: list = ["tasmax"],
+    return_type: list = 'zarr',
```

Comment:
```suggestion
    return_type: list = 'str',
```

Reply:
i think you meant to make this `str = 'zarr'`, which is great! at least that's what i'm implementing! let me know if otherwise.